puma_state_machine:
  add_pose_topic: /initialpose
  ns_topic: /puma/state_machine 
  gps_topic: /puma/sensors/gps/fix
  odometry_topic: /puma/localization/ekf_odometry
  change_mode_topic: /puma/control/change_mode

  goal_frame_id: map
  odom_frame_id: odom
  base_frame_id: base_link
  wait_duration: 4.0
  distance_tolerance: 2.0

  enable_topic: /puma/tag_detector/enable
  dis2tag_goal: 1.0
  pid:
    kp: 0.1
    ki: 0.01
    kd: 0.05
  max_vel: 0.4